packages/rancher-monitoring/prometheus-federator/generated-changes/dependencies/helmProjectOperator/dependency.yaml

```diff
@@ -1,4 +1,4 @@
 workingDir: ""
 url: https://github.com/rancher/prometheus-federator.git
 subdirectory: charts/prometheus-federator/0.1.1/charts/helmProjectOperator
-commit: b66225a673640a52407d418bb0d47d7ea94e02cc
+commit: aa0be6fdb3aa13098e3c5a2051ff664fea5726d8
```

Comment:
nit: make sure you bump https://github.com/rancher/charts/blob/4a14100b859b1d6465878fde43ca41fa104fb790/packages/rancher-monitoring/prometheus-federator/package.yaml#L3 as well